extensions/sql-migration/src/api/utils.ts

```diff
@@ -1243,3 +1243,20 @@ export async function clearDropDownWithLoading(dropDown: DropDownComponent): Pro
 	await dropDown.updateProperty('value', undefined);
 	await dropDown.updateProperty('values', []);
 }
+
+export async function promptUserForFile(filters: { [name: string]: string[] }): Promise<string> {
+	const options: vscode.OpenDialogOptions = {
+		defaultUri: vscode.Uri.file(getUserHome()!),
```

Comment:
What file will this select since getUserHome() gives a folder destination not file destination.

Reply:
It will open the file explorer and user has to manually select the file inside the folder. Selected file is used further.